lib/formBuilder.tsx

```diff
@@ -356,6 +359,3 @@ export const GenerateElement = (props: GenerateElementProps): React.ReactElement
   const generatedElement = _buildForm(element, language, t);
   return <>{generatedElement}</>;
 };
-
-export const getFormInitialValues = logger(_getFormInitialValues);
```

Comment:
The logger function emits debug level logs and error logs should there be an error. This is advisable to keep for local development and to emit errors in a production capacity 

```TS
export const logger =
  <A extends unknown[], R>(f: (...a: A) => R) =>
  (...args: A): R => {
    // Add nicer formatting for arguments being passed
    logMessage.debug(`${f.name} function called`);
    let value;
    try {
      value = f(...args);
    } catch (error) {
      logMessage.error(error as Error);
      throw error;
    }
    // Add formatting for value being returned
    logMessage.debug(`${f.name} function returned`);
    return value;
  };

export default logger;
```